frontend/src/components/TE/Versions.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <article v-for="(version, index) in versions" :key="index">
+      <h3>Version: {{ index + 1 }}</h3>
+      <p>
+        {{ formatDurationUtc(version.timestamp, "MM-DD-YYYY hh:mm a", "ms") }}
+        <a :href="version.uploadPath">File</a>
+      </p>
+      <p>
+        {{ formatDurationUtc(version.resolution.timestamp, "MM-DD-YYYY hh:mm a", "ms") }}
+        {{ version.resolution.feedback }}
+      </p>
+    </article>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Mixins } from "vue-property-decorator";
+import FormatDurationUtc from "@/mixins/FormatDurationUtc";
+
+@Component({
+  name: "Versions"
+})
+export default class Versions extends Mixins<FormatDurationUtc>(
```

Comment:
I cannot understand the reasoning for this mix-in. Please make it obvious or reconsider it.